pkg/helmcertifier/checks/registry.go

```diff
@@ -19,7 +19,8 @@
 package checks
 
 type Result struct {
-	Ok bool
+	Ok     bool
+	Reason string
```

Comment:
Would you document what `Reason` is expected to contain?

Reply:
done